pegasus/sites.v3/code.org/views/global/fa/farsiheader.haml

```diff
@@ -3,6 +3,9 @@
   user_type = request.cookies[cookie_key]
   user_type = "student" if user_type == "student_y"
 
+  # Forcibly set the 'ge_region' parameter to Farsi
+  params[:ge_region] = 'fa'
```

Comment:
This will unify the header configuration between pegasus and dashboard instead of implementing it in some bespoke way here.